packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -1789,6 +1789,17 @@ const schema: SchemaType<DbPost> = {
     hidden: true,
   },
 
+  // If the post has ever been undrafted and published
+  wasEverUndrafted: {
+    type: Boolean,
+    optional: true,
+    nullable: false,
+    ...schemaDefaultValue(false),
+    canRead: ['members'],
+    canCreate: ['members'],
+    canUpdate: ['members', 'sunshineRegiment', 'admins'],
```

Comment:
Do you want members to be able to do those things? Also there's no benefit to have the sunshine and admin groups be explicitly able to update it if members can.

Reply:
Yes - I think members is correct. Will remove the extra sunshineRegiment and admins - it's copypasta.